README.md

```diff
@@ -3,6 +3,9 @@
 [![Puppet Forge endorsed](https://img.shields.io/puppetforge/e/elastic/elasticsearch.svg)](https://forge.puppetlabs.com/elastic/elasticsearch)
 [![Puppet Forge Version](https://img.shields.io/puppetforge/v/elastic/elasticsearch.svg)](https://forge.puppetlabs.com/elastic/elasticsearch)
 [![Puppet Forge Downloads](https://img.shields.io/puppetforge/dt/elastic/elasticsearch.svg)](https://forge.puppetlabs.com/elastic/elasticsearch)
+[![Puppet Forge Score](https://img.shields.io/puppetforge/f/elastic/elastic_stack.svg)](https://forge.puppetlabs.com/elastic/elastic_stack)
+[![Build Status](https://travis-ci.org/elastic/elasticsearch.png?branch=master)](https://travis-ci.org/elastic/elasticsearch)
```

Comment:
Is this message expected when clicking on the travis link?

<img width="518" alt="Screenshot 2020-03-13 at 12 34 11" src="https://user-images.githubusercontent.com/8582351/76617627-f8780800-6526-11ea-989c-56868fc9c064.png">

Reply:
Ah, looks like I typo'd the link... Let me fix that :)